rust/main/utils/run-locally/src/sealevel/mod.rs

```diff
@@ -98,6 +98,8 @@ fn run_locally() {
         .hyp_env("DB", relayer_db.to_str().unwrap())
         .hyp_env("CHAINS_SEALEVELTEST1_SIGNER_KEY", RELAYER_KEYS[0])
         .hyp_env("CHAINS_SEALEVELTEST2_SIGNER_KEY", RELAYER_KEYS[1])
+        .hyp_env("CHAINS_SEALEVELTEST1_SUBMITTER", "Lander")
+        .hyp_env("CHAINS_SEALEVELTEST2_SUBMITTER", "Lander")
```

Comment:
I wonder if it makes sense to test situation when one chain uses `Lander` submitter while the other one `Classic`. Probably, not that easy to test since we are sending messages in one direction in E2E tests. And it does not add a lot of value since `Lander` and `Classic` submitter are independent. Only value is that we keep making sure that we did not break `Classic` submitter by implementing `Lander` submitter.

Reply:
non-svm e2e still uses classic, so that goes to show that the `Lander` integration didn't break anything. For SVM specifically, we'll need to remember to update e2e to use `Classic` if any changes to that submitter are made